src/main/groovy/org/shipkit/internal/gradle/git/GitPush.java

```diff
@@ -55,19 +55,6 @@ static void setPushUrl(GitPushTask pushTask, String writeTokenEnvValue, String g
         }
     }
 
-    public static String getWriteToken(ReleaseConfiguration conf, String writeTokenEnvValue) {
-        String token = conf.getLenient().getGitHub().getWriteAuthToken();
-        if (token != null) {
-            LOG.lifecycle("  'git push' uses GitHub write token specified in shipkit configuration.");
```

Comment:
It was kind of nice that the build log indicated how we are doing git push - with the auth token or without it. It gives the user understanding what's going on, reduces "magic" :)

We can move on for now, not a blocker!